integrations/newrelic/.port/resources/port-app-config.yaml

```diff
@@ -71,3 +52,22 @@ resources:
             error_rate: .apmSummary.errorRate
             response_time_avg: .apmSummary.responseTimeAverage
             instance_count: .apmSummary.instanceCount
+  - kind: newRelicAlert
```

Comment:
Did this actually change? if not then please make sure there is no actual diff to keep clean history

Reply:
I preferred to change the order, it makes more sense to first export the services and then the alerts